savant/app/query.py

```diff
@@ -153,3 +157,39 @@ def new_users(self):
         self.conn.commit()
         
         return self.new_users
+
```

Comment:
Following up with the meeting last Wednesday:

Please transition testing to use only `GET` and `CREATE` API endpoints for the testing. Going that route, I am guessing we don't need `null_cases` or `count_rows`. The test instead does a `CREATE` that would have triggered the null condition, then does a `GET` to verify the count of records.